### PR TITLE
Removing not used StartPartitionId

### PR DESCRIPTION
**Pull request by SergeyGaluzo**
## Description
Describe the changes in this PR.

## Related issues
Addresses [issue #].

## Testing
Describe how this change was tested.

## FHIR Team Checklist
- [ ] **Update the title** of the PR to be succinct and less than 50 characters
- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [ ] Tag the PR with Azure API for FHIR if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
- [ ] Tag the PR with Azure Healthcare APIs if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
- [ ] CI is green before merge
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj [hunk 1]
```diff
@@ -43,10 +43,6 @@
     <TSqlScript Include="Features\Schema\Sql\Tables\*.sql" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Remove="Features\Schema\Migrations\35.diff.sql" />
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Health.Fhir.Core\Microsoft.Health.Fhir.Core.csproj" />
```
Comment (SergeyGaluzo) [inline src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj:47]: This should not be needed.

File: src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Search/SortValueJObjectGenerator.cs [hunk 2]
```diff
@@ -4,10 +4,7 @@
 // -------------------------------------------------------------------------------------------------
 
 using System;
-using System.Collections.Generic;
 using System.Globalization;
```
Comment (SergeyGaluzo) [inline src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Search/SortValueJObjectGenerator.cs:7]: This was marked as errors on local build.

File: src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosResourceWrapper.cs [hunk 3]
```diff
@@ -5,12 +5,10 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using EnsureThat;
```
Comment (SergeyGaluzo) [inline src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosResourceWrapper.cs:8]: Error on local build

**Comment by SergeyGaluzo** [comment]
>     public interface IQueueClient

Not sure why it is not called JobQueueClient

---
Refers to: src/Microsoft.Health.TaskManagement/IQueueClient.cs:12 in ca92b61. [](commit_id = ca92b6153c9f5effe39e420ebb8409fa890feeec, deletion_comment = False)

**Review by brendankowitz — APPROVED** [review]

**Review by tongwu-msft — APPROVED** [review]

**Review by LTA-Thinking — APPROVED** [review]

**Pull request merged**